requirements/prod.txt

```diff
@@ -30,18 +30,15 @@ cryptography==36.0.1
     # via
     #   -r requirements/prod.in
     #   authlib
-    #   jwcrypto
-deprecated==1.2.13
-    # via jwcrypto
 docutils==0.16
     # via -r requirements/prod.in
 faker==12.0.0
     # via
     #   -r requirements/prod.in
     #   snowfakery
-fs==2.4.14
+fs==2.4.15
     # via -r requirements/prod.in
-github3.py==3.0.0
+github3-py==3.1.0
```

Comment:
😑

Reply:
I think it's from this change in pip-tools: https://github.com/jazzband/pip-tools/pull/1572

Reply:
I think it's from this change in pip-tools: https://github.com/jazzband/pip-tools/pull/1572